catalog/spatial/geocoding/spatial-geocoding-plugin/pom.xml

```diff
@@ -65,7 +72,7 @@
                             ddf.catalog.util.impl
                         </Private-Package>
                         <Embed-Dependency>
-                            commons-lang3
+                            commons-lang3, catalog-core-api-impl, platform-util
```

Comment:
`catalog-core-api-impl` has a dependency on `platform-util` Its for a single static method that is ~4 lines long. 
(Aware this is a link that some may not be able to access) https://connexta.slack.com/archives/C0B9XC1F0/p1491413170475395?thread_ts=1491408676.160564&cid=C0B9XC1F0